src/lib/components/drs/pages/DrsObjectForm.js

```diff
@@ -50,7 +53,9 @@ const DrsObjectForm = (props) => {
                     p[key] = {...props.groupedFormProps[key], readOnly:false};
                     break;
                 case FormViewType.EDIT:
-                    // TODO HANDLE EDIT
+                    nonEditableOnEditForm.some(element => element === key)
```

Comment:
although this is functionally correct, it looks a bit overcomplicated IMO, the inner `for` loop is a bit inefficient.  Consider using a `set` data structure for the `nonEditableOnEditForm`. Then you can just ask, is `key` inside my `set`?
See -> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set